### PR TITLE
Add error page (DDR-23)

### PR DESCRIPTION
**Pull request by alexissyp**

### CODE REVIEW

File: src/features/error/styled.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Typography, Image } from 'antd';
+import styled, { defaultSpacing } from '../../types/styled-components';
```
Comment (Chris-SG) [inline src/features/error/styled.ts:2]: Are we able to use `types/styled-components` here?

File: src/features/error/index.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Space, Typography } from 'antd';
+import logo from 'assets/logo.png';
+import { ErrorWrapper, ErrorHeader, LargeWidthImage } from './styled';
+
+const Error = () => {
+  const { Title, Link } = Typography;
```
Comment (Chris-SG) [inline src/features/error/index.tsx:7]: I'm unsure if this should be inside the `Error` function or outside. Either works, but I'm going to ping @nhivn for input here.

**Review by Chris-SG — CHANGES_REQUESTED** [review]
Awesome job! 2 small comments :)

File: src/features/error/index.tsx (hunk 2 above)
Comment (alexissyp) [reply src/features/error/index.tsx:7]: Hello. Any update on this? :)

Comment (Chris-SG) [reply src/features/error/index.tsx:7]: Sorry! Yes we should bring it out. Once thats done I can approve :)

**Review by Chris-SG — APPROVED** [review]

**Pull request merged**